hr_timesheet_overview/models/hr_employee.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class Contract(models.Model):
+    _inherit = "hr.employee"
+
+    last_hours_report_date = fields.Date(
```

Comment:
```suggestion
   hours_report_last_update = fields.Date(
```
Better to understand ?